### PR TITLE
Improve colors preview one-liner

### PR DESCRIPTION
**Pull request by camsteffen**
I think this code looks a lot better for previewing colors. Ok so I got bored.

### CODE REVIEW

**Comment by romkatv** [comment]
This no longer shows foreground colors, which many users need (*Lean* and *Pure* styles have no background colors but do have foreground colors).

The line break after the first 4 colors looks odd.

What are you trying to improve?

File: README.md [hunk 1]
```diff
@@ -849,7 +849,7 @@ Type `source ~/.p10k.zsh` to apply your changes to the current Zsh session.
 To see how different colors look in your terminal, run the following command:
 
 ```zsh
-for i in {0..255}; do print -Pn "%K{$i} %k%F{$i}${(l:3::0:)i}%f " ${${(M)$((i%8)):#7}:+$'\n'}; done
+for i in {0..255}; do print -Pn "%K{$i} %F{0}${(l:3::0:)i}%f %k"${${(M)$((i%6)):#3}:+$'\n'}; done
```
Comment (camsteffen) [inline README.md]:
```suggestion
for i in {0..255}; do print -Pn "%K{$i}  %k%F{$i}${(l:3::0:)i}%f " ${${(M)$((i%6)):#3}:+$'\n'}; done
```

**Comment by camsteffen** [comment]
The line break there makes the colors arranged so that related colors are in the same row/column.

**Comment by romkatv** [comment]
> The line break there makes the colors arranged so that related colors are in the same row/column.

Oh, that's nice! Thanks!

**Pull request merged**